Dockerfile.build

```diff
@@ -42,11 +43,12 @@ RUN add-apt-repository "$LLVM_REPO_NAME"
 RUN apt-get update && \
     apt-get install -y --no-install-recommends clang-format clang
 
-ARG BAZEL_VERSION
-RUN wget https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
-    chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
-    ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh && \
-    rm -f bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
+ARG USE_BAZEL_VERSION
+ARG BAZELISK_VERSION
+RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v${BAZELISK_VERSION}/bazelisk-linux-amd64 && \
+    mv bazelisk-linux-amd64 /usr/local/bin/bazel && \
+    chmod +x /usr/local/bin/bazel && \
+    bazel version
```

Comment:
Use `bazel version` to check the version and download the bazel binary when building the base container image.